README.md

```diff
@@ -1 +1,18 @@
-# Hello World
+# Hi Folks!
+
+This readme is for the .github repo *but* the .github repo operates a little differently than most repos. The Readme that folks see when they arrive at [CAWaterBoardDataCenter](https://github.com/CAWaterBoardDataCenter/) **isn't this one!** They are seeing the README.md in the profile folder. [Check it out](https://github.com/CAWaterBoardDataCenter/.github/blob/main/profile/README.md) and place your edits there if appropriate.
+
+## What is the purpose of this README?
+**Top level SWRCB GitHub Documentation** ... probably
+Also, this is open to discussion. 
+
+### Things that might go in this readme or in this repo:
+1. Open data policy documentation
+2. Water Data formats
+3. Data Governanace guiding pricipals
+4. Data Ethic documentation
+5. Github Managment roles & responsibilities
+6. data use licenses
+7. Default Account permissions for WaterBoard members and outside collaborators
+8. Justification for creation of Repos
+9. Actions to be taken after the creation of repos.
```

Comment:
Don't forget to add the Water DC urban supplier formats

Reply:
Also, include a document that describes useful concepts like: 
1. How to write useful comments
2.

Reply:
add water DC urban supplier data format language
added guidelines section to encompass future docs like "how to write useful comments" and "how much code shoudl be represented by a single commit"